gadget/layout.go

```diff
@@ -75,6 +75,11 @@ type PartiallyLaidOutVolume struct {
 // LaidOutStructure describes a VolumeStructure coming from the gadget plus the
 // OnDiskStructure that describes how it would be applied to a given disk and
 // additional content used when writing/updating data in the structure.
+//
+// Note that we need to be careful while using the fields in OnDiskStructure as
+// some times LaidOutStructure is created before we have information about the
```

Comment:
thanks, maybe adding "(for example StartOffset)" somewhere in this?

Reply:
Done